lib/whitehall_importer.rb

```diff
@@ -1,6 +1,28 @@
 # frozen_string_literal: true
 
 module WhitehallImporter
+  def self.create_migration(organisation_content_id, document_type)
+    ActiveRecord::Base.transaction do
+      whitehall_migration = WhitehallMigration.create!(organisation_content_id: organisation_content_id,
+                                 document_type: document_type,
+                                 start_time: Time.current)
+
+      whitehall_export = GdsApi.whitehall_export.document_list(organisation_content_id, document_type)
+
+      whitehall_export.each_entry do |page|
```

Comment:
From googling about what #each_entry is I don't think we need to use it instead of each: https://stackoverflow.com/questions/19840059/is-there-a-difference-between-enumerableeach-and-enumerableeach-entry-in-ruby

Reply:
Good spot, I've updated this to use `each`